core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -190,6 +193,27 @@ class GroupCoordinatorTest extends JUnitSuite {
     assertEquals(Errors.NOT_COORDINATOR, joinGroupError)
   }
 
+  @Test
+  def testJoinGroupShouldReceiveErrorIfGroupOverMaxSize() {
```

Comment:
seems to be consistently failing for me with: 
java.util.concurrent.TimeoutException: Futures timed out after [1 millisecond]

Reply:
I do not see any failures. Passed 100/100.